Test/BillingInfoTest.cs

```diff
@@ -78,7 +78,7 @@ public void UpdateBillingInfoWithToken()
             {
                 billingInfo.Update();
             }
-            catch (NotFoundException exception)
+            catch (ValidationException exception)
```

Comment:
This should have been changed in the 1.15.8 release, but it wasn't so I went ahead and fixed it now